README.org

```diff
@@ -55,6 +52,37 @@ The package provides 6 classes for accessing ZLib streams:
 
 For all stream objects, the =badbit= of their expection mask is turned on in order to propagate exceptions.
 
+**** CMake
+
+There are two simple ways to add zstr to a CMake project.
```

Comment:
three

Reply:
I'll update now, good catch!

Reply:
I added Method 3. 
I dropped the lines relating to setting a cache variable, as that's more an implementation detail that can be left up to the user.